Source/Source/JoyGiver_BuyStuff.cs

```diff
@@ -228,20 +247,4 @@ public static bool CanEat(Thing thing, Pawn pawn)
     {
         return thing.def.IsNutritionGivingIngestible && thing.def.IsWithinCategory(ThingCategoryDefOf.Foods) && ItemUtility.AlienFrameworkAllowsIt(pawn.def, thing.def, "CanEat");
     }
-
-    private bool HasRecentlyLookedAt(Pawn pawn, IntVec3 cell)
```

Comment:
This was there so they don't look at the same stuff all the time. I assume this will happen again now?